src/snovault/viewconfigs/base_view.py

```diff
@@ -44,6 +52,34 @@ def __init__(self, context, request):
         from_, page_size = get_pagination(request)
         self._from_ = from_
         self._size = page_size
+        self._view_name = '' # should be set by view
```

Comment:
@keenangraham Good point. I set it to empty string because children (technically mostly grandchildren) classes need this to be a string if they do not set it themselves and None is not really a string. Nevertheless, I can set it to None or make and _factory_name have a default string-value.

Reply:
I agree.  I think best practice is to initialize as None and then code to that standard.   But it's possible that's is a lot of work given our code base.  @zoldello If you can change it to None without rewriting a bunch of stuff then do so.

Reply:
Done.